src/main/java/yevhenii/lostfilmdemo/convertors/TVSeriesConvertor.java

```diff
@@ -0,0 +1,60 @@
+package yevhenii.lostfilmdemo.convertors;
+
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.stereotype.Component;
+import yevhenii.lostfilmdemo.entity.FeedMessage;
+import yevhenii.lostfilmdemo.entity.TVSeries;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@Component
+public class TVSeriesConvertor implements Converter<FeedMessage, TVSeries> {
+
+    //TODO instead of all this methods which are parsing String use regex expression with group for each value you wanna to extract
+    private String getName(String regexp, String input){
+
+        Pattern pattern = Pattern.compile(regexp);
```

Comment:
compiling regex on a fly is to expensive, extract it to static class variable to do it once, regex won't change so there is no reason to do it inside the method